src/task/src/pullRequests/pullRequest.ts

```diff
@@ -82,6 +86,10 @@ export default class PullRequest {
   public async getUpdatedTitle (currentTitle: string): Promise<string | null> {
     this._logger.logDebug('* PullRequest.getUpdatedTitle()')
 
+    if (!this._inputs.changeTitle) {
+      return null
+    }
```

Comment:
Thanks for adding this early in the method. I like this because it skips all of the long-running checks for those cases where the title should not be set.